src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer has started working");
+    }
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working");
+    }
```

Comment:
```suggestion
    public void doWork() {
        System.out.println("Bulldozer has started working");
    }
    
    public void stopWork() {
        System.out.println("Bulldozer has stopped working");
    }
```

There must be an empty line between methods